fdm-core/src/authorization.ts

```diff
@@ -500,17 +500,20 @@ async function getResourceChain(
             const result = await fdm
                 .select({
                     farm: schema.fieldAcquiring.b_id_farm,
-                    field: schema.fieldSowing.b_id,
+                    field: schema.cultivationStarting.b_id,
                     cultivation: schema.cultivations.b_lu,
```

Comment:
in changelog you mention cultivationstarting, and here you use cultivationStarting. Does use of capitals imply different object/value?

Reply:
good to know. thanks